### DISCUSSION TITLE
Please enable me to opt out from the achievements feature.

### DISCUSSION THREAD

**Discussion by brittanybelle** [community/community#27533] [Other Feature Feedback, Questions, & Ideas] [answered]
I very strongly do not want to show achievements in my profile (to anyone). I additionally do not want to see any achievements anywhere on others' profiles.

The achievements system sets up terrible incentives for folks to create PRs not for the purpose of making a meaningful contribution, but simply to get another badge checked off their list. I think this entire feature is terrible and I very strongly want to opt out.

**Comment by radar-blip** [comment, marked as answer]
There is a checkbox in settings to not show achievements.
![Screenshot from 2022-07-28 18-08-47](https://user-images.githubusercontent.com/39196767/181645799-5303b5bd-0ba0-4ef8-8a7e-86d2eb8c158c.png)

**Reply by brittanybelle** [reply]
Thank you for pointing this out to me.

**Reply by radar-blip** [reply]
No problem. Hope it helps. 👍🏼

**Comment by DumbB1** [comment]
I would Greatly Appreciate if I Could Respectfully OPT-OUT PLEASE I DO NOT WISH TO BE APART OF YOUR DEVELOPER PROJECT!!!!

**Reply by zaahiral** [reply]
there should be a setting in profile settings to untick shown achievements on your profiles

**Comment by zaahiral** [comment]
there should be a setting in profile settings to untick shown achievements on your profiles
